src/stan-users-guide/odes.Rmd

```diff
@@ -1,11 +1,41 @@
 # Ordinary Differential Equations  {#ode-solver.chapter}
 
-Stan provides a built-in mechanism for specifying and solving systems
-of ordinary differential equations (ODEs). Three different solvers,
-tuned for solving non-stiff systems and for stiff systems are available.
+Stan provides four different methods for solving systems of ordinary
+differential equations (ODEs). All of these methods will adaptively
+refine their solutions in order to satisfy the given tolerances.
+
+The upside to this is that as long as reasonable tolerances are provided
+and an ODE solver well-suited to the problem is chosen, the choice of timestep
+can be abstracted away. The downside is that it is not always clear from the
+outset what reasonable tolerances are, or which ODE solver is best suited
+to each problem. Additionally, in the course of working with an ODE model in
+Stan the correct answer may change (for instance, the ODE may change).
```

Comment:
This is a non-issue. If model changes, methodology should change accordingly, as in the other parts of Stan model.

Reply:
> methodology should change accordingly

Yeah, and this is trying to make it clear that if you're using ODEs changing the solver is part of the methodology. It's entirely possible someone might not think of their ode solver as a critical choice (or know a methodology to pick one).